buildlog/buildlog_test.go

```diff
@@ -0,0 +1,198 @@
+package buildlog
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"regexp"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	intervalTime = 500
+	timeRegEx    = `[\d]{4}-[\d]{2}-[\d]{2} [\d]{2}:[\d]{2}:[\d]{2}`
+)
+
+var (
+	testInputs = []string{
+		`{"t": 1581662022394, "m": "test 1", "n": 0, "s": "main"}` + "\n",
+		`{"t": 1581662022395, "m": "test 2", "n": 1, "s": "main"}` + "\n",
+	}
+)
+
+type writeCloser struct {
```

Comment:
we can delete this struct and `Close` function

Reply:
I fixed it.